{{cookiecutter.project_slug}}/app/tasks/__init__.py

```
@@ -0,0 +1 @@
+from .ping import *  # noqa: F403
```

Flake8 (7.2.0)

[error] '.ping.*' imported but unused

(F401)